### ISSUE TITLE
[Android] TalkBack accessibility using markers

### ISSUE THREAD

**Issue by Ferdzz** [mapbox/mapbox-gl-native#16128]
This issue discusses the availability of TalkBack accessibility features as used within the Mapbox Android SDK. It touches feature parity with the Mapbox iOS SDK.

Mapbox on iOS seems to be integrated with the VoiceOver accessibility features, allowing the user to swipe left and right to select the previous / next annotation markers. The user is then able to double tap to perform the "click" action on that marker. That feature appears to be missing on the Android SDK. When I attempt to swipe through the markers with TalkBack enabled on Android, it behaves as if the markers did not exist as accessibility-enabled UI elements.

I have come across another issue (#4857) touching the same topic from 2017, saying that it was fixed & implemented in v5.0.0. That didn't seem to be enough since we are running on v6.8.1

I wasn't able to find any documentation on how to enable the accessibility features. Would it be possible to help us find some more information on how to make the map accessible to the visually impaired?

**Comment by tobrun** [comment]
We are tracking this feature in https://github.com/mapbox/mapbox-plugins-android/issues/957

**Comment by Ferdzz** [comment]
Thank you, we will keep an eye out on that issue
